eng/Build.props

```diff
@@ -12,8 +12,8 @@
     <!-- Exclude VSIX projects on non-Windows -->
     <ProjectToBuild Condition="'$(OS)'!='WINDOWS_NT' and '$(SdkTaskProjects)'==''" Include="$(MSBuildThisFileDirectory)..\src\Razor\Razor.Slim.sln" />
 
-    <NodeJsProjects Include="$(RepoRoot)src\Razor\src\**\*.npmproj" RestoreInParallel="false"/>
-    <NodeJsProjects Include="$(RepoRoot)src\Razor\test\**\*.npmproj" RestoreInParallel="false" />
+    <NodeJsProjects Include="$(RepoRoot)src\Razor\src\**\*.npmproj" Exclude="$(RepoRoot)src\**\node_modules\**" RestoreInParallel="false"/>
```

Comment:
What does this do?

Reply:
I don't know why nobody else runs into this, but without this change I get weird failures after I build where it's trying to restore things INSIDE a node_modules folder. Mutating the insides of a node_modules folder is a no-no so I've ecluded any nod_modules from returning results.

Reply:
Thanks for fixing this, I just ran into this here: https://github.com/dotnet/aspnetcore/issues/28230

Reply:
I'm just happy to learn that I'm not insane and this DOES happen to other people lol.